src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -137,8 +138,23 @@ export class HuiCardPicker extends LitElement {
       return;
     }
 
-    this._unusedEntities = computeUnusedEntities(this.hass, this.lovelace);
-    this._usedEntities = [...computeUsedEntities(this.lovelace)];
+    this._unusedEntities = computeUnusedEntities(
+      this.hass,
+      this.lovelace
+    ).filter(
+      (eid) =>
+        this.hass!.states[eid] &&
```

Comment:
All entities returned by unused entities exist on `hass.states`

Reply:
You would think. Getting errors with out that

Reply:
Oh yeah never mind. Used Entities takes everything from your lovelace, which can contain unavailable ones too.